esmvaltool/recipe_schema.yml

```diff
@@ -12,8 +12,8 @@ diagnostics: map(include('diagnostic'), required=False)
 dataset:
   dataset: str()
   project: str(required=False)
-  start_year: int(required=False, min=0000, max=4000)
-  end_year: int(required=False, min=0000, max=4000)
+  start_year: int(min=0000, max=5000)
```

Comment:
Can you put `required=False, ` back? The `start_year` and `end_year` can also be specified in the variable section(s), so in that case they are not required in the dataset section(s).